_includes/releases/babelfish-1.1.1.md

```diff
@@ -0,0 +1,111 @@
+
+- Babelfish Version: 1.1.1
+- PostgreSQL Server Version: 13.5
+- Download source distributions:
+  - [BABEL_1_1_1__PG_13_5.zip](https://github.com/babelfish-for-postgresql/babelfish-for-postgresql/releases/download/BABEL_1_1_1__PG_13_5/BABEL_1_1_1__PG_13_5.zip)
+  - [BABEL_1_1_1__PG_13_5.tar.gz](https://github.com/babelfish-for-postgresql/babelfish-for-postgresql/releases/download/BABEL_1_1_1__PG_13_5/BABEL_1_1_1__PG_13_5.tar.gz)
+- Babelfish Compass
+  - [Download](https://github.com/babelfish-for-postgresql/babelfish_compass/releases)
+- Date: November 8, 2022
+
+## Overview
+
+Babelfish 1.1.1 contains all of the features and limitations of version 1.1.0:
+
+* Enhancements and fixes that improve the user experience, listed below.
+* Additional built-in functions, including `ORIGINAL_LOGIN()`, `SQUARE()` and more.  
+* Additional system views, including `sys.columns`, `sys.sysindexes`, and more.
+* Additional system stored procedures, including `sp_columns`, `sp_statistics`, and more.
+
+The above items and other new features are explained in more detail in the sections below.
+
+## Upgrading to the new version
+
+**If you’re running Babelfish in a managed environment, please check your vendor’s instructions for upgrading.**
+
+Babelfish 1.1.1 is compatible with the latest major version; a dump/restore is not required to upgrade to the newer version. To upgrade:
+
+1. Install the new binaries; for details, see [Compiling Babelfish From Source] (https://babelfishpg.org/docs/installation/compiling-babelfish-from-source/).
+
+2. Start the new server.
+
+3. Update the modified extensions. 
+
+To update the extensions that have changed since the last release, connect to the PostgreSQL endpoint of the Babelfish database and execute the following statements from a single session in this order:
+
+```bash
+ALTER EXTENSION "babelfishpg_common" UPDATE;
+ALTER EXTENSION "babelfishpg_tsql" UPDATE;
+```
+
+Please note that the extension owner must invoke ALTER EXTENSION. For this release, the `babelfish_tds` and `babelfish_money` extensions don’t include any SQL-level changes and need not be updated.
+
+After the extensions are updated, you can check the extension version from the PostgreSQL port:
+
+```bash
+babelfish_db[124386]# \dx
+                                List of installed extensions
+        Name        | Version |   Schema   |                   Description
+--------------------+---------+------------+-------------------------------------------------
+ babelfishpg_common | 1.1.1   | public     | Transact SQL Datatype Support
+                                 ....
+ babelfishpg_tsql   | 1.1.1   | public     | Transact SQL Compatibility
```

Comment:
The individual extension versions are not always same as Babelfish version, here in this case the above versions should be `1.1.0` instead of `1.1.1`. 
Additionally, we should provide the output of below command for checking if Babelfish upgrade was successful:
```
babelfish_db[124386]# SELECT serverProperty('BabelfishVersion') AS BabelfishVersion;
 babelfishversion 
------------------
 1.1.1
(1 row)
```

Reply:
Updated...